### DISCUSSION TITLE
Authentication Error 403 (Forbidden)

### DISCUSSION THREAD

**Discussion by WijdanMTak** [firebase/flutterfire#5327] [Q&A]
I am developing an android app using flutter, I am trying to add social login to the app using firebase authentication, I followed all the steps required to integrate the app with firbaise, and  FirebaseApp initialization successful ...
But I am facing a "Your client does not have permission to get URL " problem when trying to sign in using Facebook or Google, Although the progress work **successfully** when I try it with **VPN**, so can I have some help please to solve this issue.

` E/com.google.android.gms.internal.firebase-auth-api.zzvy: Failed to parse error for string [<!DOCTYPE html><html lang=en>  <meta charset=utf-8>  <meta name=viewport content="initial-scale=1, minimum-scale=1, width=device-width">  <title>Error 403 (Forbidden)!!1</title>  <style>    *{margin:0;padding:0}html,code{font:15px/22px arial,sans-serif}html{background:#fff;color:#222;padding:15px}body{margin:7% auto 0;max-width:390px;min-height:180px;padding:30px 0 15px}* > body{background:url(//www.google.com/images/errors/robot.png) 100% 5px no-repeat;padding-right:205px}p{margin:11px 0 22px;overflow:hidden}ins{color:#777;text-decoration:none}a img{border:0}@media screen and (max-width:772px){body{background:none;margin-top:0;max-width:none;padding-right:0}}#logo{background:url(//www.google.com/images/branding/googlelogo/1x/googlelogo_color_150x54dp.png) no-repeat;margin-left:-5px}@media only screen and (min-resolution:192dpi){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat 0% 0%/100% 100%;-moz-border-image:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) 0}}@media only screen and (-webkit-min-device-pixel-ratio:2){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat;-webkit-background-size:100% 100%}}#logo{display:inline-block;height:54px;width:150px}  </style>  <a href=//www.google.com/><span id=logo aria-label=Google></span></a>  <p><b>403.</b> <ins>That’s an error.</ins>  <p>Your client does not have permission to get URL <code>/identitytoolkit/v3/relyingparty/verifyAssertion</code> from this server.  <ins>That’s all we know.</ins>] with exception: Value <!DOCTYPE of type java.lang.String cannot be converted to JSONObject`

**Comment by alexaung** [comment]
@WijdanMTak I got same issue. May I know how to do you solve it.

**Reply by hortigado** [reply]
Hi, can you solved it? i have the same error.
